dashboard/app/models/ability.rb

```diff
@@ -63,6 +64,7 @@ def initialize(user)
 
       if user.teacher?
         can :manage, Section, user_id: user.id
+        can :read, Section, user_id: user.id
```

Comment:
`:manage` means [all actions](https://github.com/ryanb/cancan/wiki/defining-abilities#the-can-method), so what does `:read` add here that isn't covered in the above?